src/containers/Classroom.jsx

```diff
@@ -8,9 +8,10 @@ import { default as ClassroomPresentational } from '../presentational/Classroom.
 export default class Classroom extends Component {
 
   render() {
+    const included = this.props.classrooms.included;
     const classroom = this.props.classrooms.data.find(classroom =>
       classroom.id === this.props.params.classroomId);
-    return (<ClassroomPresentational data={classroom} />);
+    return (<ClassroomPresentational data={classroom} included={included} />);
```

Comment:
Minor comment: the code in your update passes the sanity check very well, but the `included` variable isn't quite descriptive of its purpose. "What is `included`?" doesn't seem too clear to me; perhaps a brief //comment would help clarify it?